src/pages/comics.tsx

```diff
@@ -1,9 +1,50 @@
-import React from 'react';
+import React from "react";
+import { RouteChildrenProps } from "react-router-dom";
+import { IMarvelEntityResponse } from "../interface/interface";
+import { ComicsItem } from "../components/comicsComics/comicsItem";
+import { ComicsLogo } from "../components/comicsLogo/comicsLogo";
+import "./comics.css";
+import { LoadingOverlay } from "../components/loadingOverlay/loadingOverlay";
+import { handleRequest } from "../api/requestHandler";
 
-export class Comics extends React.Component {
-    render() {
+interface IComicsState {
+    comics: IMarvelEntityResponse[];
+    isLoading: boolean;
+}
+
+export class Comics extends React.Component<RouteChildrenProps, IComicsState> {
+    constructor(props: RouteChildrenProps) {
+        super(props);
+        this.state = { comics: [], isLoading: false };
+    }
+
+    getComics(): void {
+        this.setState({ isLoading: true });
+        handleRequest(`characters/${(this.props.match?.params as any).id}/comics`)
```

Comment:
`characters/${(this.props.match?.params as any).id}/comics` not a good way to pass params that way